tests/run/test_run.py

```diff
@@ -142,6 +142,32 @@ def test_local_runtime_failure_before_executing_the_function_code(db):
     assert "failed on pre-loading" in str(exc.value)
 
 
+def test_local_runtime_with_kwargs(db):
```

Comment:
would be nice if you could add some more test cases for example:
1. if kwargs is empty
2. if params is empty
3. if param x is in kwargs